include/Configuration.h

```diff
@@ -55,6 +56,8 @@ struct CONFIG_T {
     char Mqtt_Password[MQTT_MAX_PASSWORD_STRLEN + 1];
     char Mqtt_Topic[MQTT_MAX_TOPIC_STRLEN + 1];
     bool Mqtt_Retain;
+    bool Mqtt_Tls;
```

Comment:
The type should only be extended at the end, because otherwise settings get lost for existing configs

Reply:
done